.travis.yml

```diff
@@ -7,59 +7,69 @@ branches:
     - main
 cache:
   directories:
-    - $HOME/docker
-env:
-  global:
-    - DOCKER_IMAGE='cerner/terra-application'
-    - TAG="${TRAVIS_PULL_REQUEST_SHA:=latest}"
+    - travis-build
 before_install:
-  - sudo chown -R travis:travis $HOME/docker
-  # Load cached docker images
-  - if [[ -d $HOME/docker ]]; then ls $HOME/docker/*.tar.gz | xargs -I {file} sh -c "zcat {file} | docker load"; fi
-before_cache:
-  - sudo chown -R travis:travis $HOME/docker
-  - if [ $TRAVIS_TEST_RESULT == 1 ]; then rm -f $HOME/docker/*.tar.gz; fi
+  # Clean install on any new build
+  - npm run clean
 jobs:
   include:
-    - stage: build
-      before_install: true
+    - stage: lint jest and compile
       script:
-        - sudo chown -R travis:travis $HOME/docker
-        - 'rm -f $HOME/docker/*.tar.gz'
-        - travis_wait docker-compose build test-ci
-        - travis_wait docker-compose --verbose pull standalone-chrome
-        - >
-          mkdir -p $HOME/docker && docker images -a --filter='dangling=false' --format '{{.Repository}}:{{.Tag}} {{.ID}}'
-          | xargs -n 2 -t sh -c 'test -e $HOME/docker/$1.tar.gz || docker save $0 | gzip -2 > $HOME/docker/$1.tar.gz'
-    - stage: lint and jest
-      script:
-        - "travis_wait docker-compose run -e HAS_JOSH_K_SEAL_OF_APPROVAL=$HAS_JOSH_K_SEAL_OF_APPROVAL -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG test-ci sh -c 'npm run lint && npm run jest'"
+        # clear the cache of any old build artifacts
+        - rm -rf ./travis-build
+        - npm run lint
+        - npm run jest
+        # These can't be built in parallel because the cache is a single object and you can't add to it in parallel
+        - npm run compile:prod -- --env.disableAggregateThemes --output-path $TRAVIS_BUILD_DIR/travis-build/default
+        - npm run compile:prod -- --env.defaultLocale=de --env.disableAggregateThemes --output-path $TRAVIS_BUILD_DIR/travis-build/default-locale
+        - npm run compile:lowlight -- --output-path $TRAVIS_BUILD_DIR/travis-build/lowlight
+        - npm run compile:fusion -- --output-path $TRAVIS_BUILD_DIR/travis-build/fusion
+    # Split out all the wdio runs to not eat up all the travis executors at once.
+
     - stage: wdio
+      name: default-theme
+      # Don't clean install
+      before_install: skip
+      # Rely on the cache for node_modules.. this won't work for any node modules in the packages dir but works for running wdio
+      install: skip
+      script:
+        # we start up our own standalone chrome instance because the Selenium Docker Service doesn't seem to work here.
+        - docker-compose up -d standalone-chrome-travis
+        # using the travis env section to set env variables seems to break cache
+        - SITE=$TRAVIS_BUILD_DIR/travis-build/default npm run wdio-default
+    - name: default-locale
```

Comment:
nit: default-theme-de-locale

Reply:
we need to move this away from de which is why i named it default-locale... we're testing setting the default locale.... at least that was my thinking